src/components/Header/Header.tsx

```diff
@@ -3,12 +3,26 @@ import styles from './Header.module.scss';
 import logo from '../../assets/img/logo.png';
 import { Path } from '../../pages/constants/path';
 import Navigation from '../Navigation/Navigation';
+// import InputField from '../../UI/InputField';
 
 export default function Header() {
   const { pathname } = useLocation();
 
   return (
     <header className={styles.header}>
+      {/* just for test */}
```

Comment:
This commented code don't needed. You can use this in your branch, but don't add in production version.